src/components/jurisdiction/Jurisdiction.vue

```diff
@@ -41,15 +42,26 @@ export default class Jurisdiction extends Vue {
         separator: (jur.value === JurisdictionLocation.FD)
       }))
 
+    if (this.showUsJurisdictions) {
+      array.push({ isHeader: true, group: 1, text: 'United States' })
+      UsaJurisdiction
+        .forEach(jur => array.push({
+          group: 1,
+          text: jur.text,
+          value: jur.value,
+          separator: false
+        }))
+    }
+
     // add in International jurisdictions (not including CA)
-    array.push({ isHeader: true, group: 1, text: 'International' })
+    array.push({ isHeader: true, group: 2, text: 'International' })
     IntlJurisdictions
       .filter(jur => jur.value !== JurisdictionLocation.CA)
```

Comment:
Did you find out why you don't need to filter out USA when `showUsJurisdictions` is True?

Reply:
No not yet. Just wondering if somehow it is removed in `filter` or `forEach` function considering duplicates in the Array. (one added when adding USA jurisdiction and other in International jurisdiction list)

Reply:
Those filter and forEach calls are separate, and working on separate lists. I dunno. Might look at it this afternoon for fun.